emperor/support_files/js/view.js

```diff
@@ -67,11 +80,23 @@ function DecompositionView(decomp, asPointCloud) {
    * @type {THREE.Mesh[]}
    */
   this.tubes = [];
+
   /**
    * Array of THREE.Mesh objects on screen (represent samples).
    * @type {THREE.Mesh[]}
    */
   this.markers = [];
+
+  /**
```

Comment:
Would you mind including the `@type` documentation for these two properties?

Reply:
Done